guides/Getting-Started.md

```diff
@@ -16,48 +16,48 @@ The Purescript compiler (`purs`) can be installed with npm:
 
 #### Setting up the Development Environment
 
-PureScript's core libraries are configured to use the [Pulp](https://github.com/purescript-contrib/pulp) build tool, and packages are available in the [Bower registry](http://bower.io/search/?q=purescript-).
+PureScript's core libraries are configured to use the [Spago](https://github.com/spacchetti/spago) package manager and build tool.
```

Comment:
If the move to `purescript/spago` will happen quite soon we need to make sure to update this link (though fortunately GitHub will redirect automatically in the worst case).

Reply:
Yes, I was thinking about this as I pasted the link. That said, the update should be speedy (and, as you said, the redirect will work in the interim).